source/getting-started/index.md

```diff
@@ -2,130 +2,181 @@
 
 Try an example of sending a message via a NodeX Agent and receiving it.
 
-## Installing NodeX Agent and set up
+## Set up NodeX Agent
```

Comment:
IMO, there are two types of installation procedures: one for source code and another for binaries. Correspondingly, there are two methods for setup. However, since the installation and setup procedures are separated, it might feel difficult to follow.

Would it be better to include the setup within the installation section? This approach could streamline the documentation and make it more user-friendly by providing a cohesive step-by-step guide.

Reply:
@masa256k1 
Thank you for your comment.

How about the following policy?

- Include setup instructions in Installation.
- Define the completion of the setup as the completion of config and network settings.
- The setup procedure should be divided into two types: one using binaries and the other using source code.